Source/WebGPU/WGSL/TypeDeclarations.rb

```diff
@@ -1075,9 +1075,17 @@
     must_use: true,
 
     [T < ConcreteInteger, U < ConcreteInteger, S < Concrete32BitNumber].(texture_1d[S], T, U) => vec4[S],
+    [F, T < ConcreteInteger, U < ConcreteInteger, S < Concrete32BitNumber].(texture_storage_1d[F, read], T) => vec4[ChannelFormat[F]],
```

Comment:
Not sure if there is a less verbose way, I see some declarations use `AM` for access mode, but these should only work for 2 of the 3 access modes. 

With this approach, no extra validation is needed in the source files